cmd/performance-profile-creator/cmd/root.go

```diff
@@ -506,8 +506,8 @@ func getProfileData(args ProfileCreatorArgs, cluster ClusterData) (*ProfileData,
 	case defaultLatency:
 		if profileData.rtKernel {
 			return nil, fmt.Errorf(
-				"please use one of %v power consumption modes together with the real-time kernel",
-				validPowerConsumptionModes[1:],
+				"%v power consumption mode is not available with real-time kernel, please use one of %v modes",
+				validPowerConsumptionModes[0], validPowerConsumptionModes[1:],
```

Comment:
if validPowerConsumptionModes[0] is always `defaultLatency`  maybe hardcode this value in the error message or use enums instead of integer indexes ?  (enums can be used for future flexibility)